Propagation-Scripts/Download-and-import.md

```diff
@@ -0,0 +1,29 @@
+﻿# Download and import Devolutions provided Template
+
+## Download a template file
+
+- Go to the folder [Propagation-Scripts/templates](./templates)
+- Get the template corresponding to your needs
+
+As much as possible, you should find specific assistance for each of the available templates referenced on the template help page.
```

Comment:
Je retirerais le "as much as possible" et indiquer "You will find specific..." Ca laisse sous entendre que l'on ne documentera pas toujours. Ce qui ne devrait pas être le cas ;-)

Reply:
C'etait le but :D je vais changer ca :)